command/command.c

```diff
@@ -289,21 +295,71 @@ process_command_output (CommandApplet *command_applet, gchar *output)
     }
 }
 
+static void
+subprocess_callback (GObject *source, GAsyncResult *result, gpointer data)
+{
+    GSubprocess *subprocess;
+    CommandApplet *command_applet;
+    GInputStream *in;
+    gsize bytes_read;
+    gchar *output;
+    GError *error = NULL;
+
+    command_applet = (CommandApplet *) data;
+    subprocess = G_SUBPROCESS (command_applet->subprocess);
```

Comment:
Why do you use `G_SUBPROCESS` here?

Reply:
I use source as gsubprocess before, now G_SUBPROCESS not needed.